### ISSUE TITLE
Vertical layout?

### ISSUE THREAD

**Issue by helske** [AdeelK93/collapsibleTree#39]
It would be nice to have option to flip the tree into top-to-bottom format.

**Comment by GRMK11** [comment]
Yes ,Hi Adeel is it possible  for vertical layout in collapsible tree/summary to include and also any option for assigning position of labels  on  or above links rather than dissecting labels by the links  connecting the node for better visibility of labels.
Thanks in Advance,
Manohar
